### ISSUE TITLE
Lorax's Tale not visible as available quest on world map (Blacksmith only quest)

### ISSUE THREAD

**Issue by EnterpriseWoW** [Questie/Questie#3572]
## Description

No Quest indicator exists for Quest ID 5126 ["Lorax's Tale"](https://tbc.wowhead.com/quest=5126/loraxs-tale) offered by [Lorax](https://tbc.wowhead.com/npc=10918/lorax) [NPC ID: 10918] in Winterspring. 

No Quest indicator:
![Winterspring](https://user-images.githubusercontent.com/108195970/175791124-75eea7fc-f035-4cba-9bb2-613be0b8e396.png)

No "Exclamation mark" at NPC:
![Lorax](https://user-images.githubusercontent.com/108195970/175791154-0d392f38-1847-4e5d-8a5f-0d1b82d4bf26.jpg)

Offers quest: 

![Quest_offer_Lorax](https://user-images.githubusercontent.com/108195970/175791183-8e03ffbb-9423-45a6-a06e-d271d6c9eaf8.jpg)

I checked the QuestDB and saw {164,285} (Blacksmith,285 minimum skill level) for the requiredSkill but I meet this requirement:

![proffession_requirement](https://user-images.githubusercontent.com/108195970/175791260-1e3c2a91-5f0d-4f18-ac82-dc9517cc927a.jpg)

## Screenshots

The Advanced Search indicates not hidden (by Questie) or Complete

![advanced_Search](https://user-images.githubusercontent.com/108195970/175791428-59f701a8-3825-4409-8b5a-db1237a0956d.jpg)

But it's not listed in the Blacksmithing Journey and Winterspring zone Journey:

![Blacksmithing_Journey](https://user-images.githubusercontent.com/108195970/175791498-82c122f6-0880-43d0-9152-7f81fdada719.jpg)

![Winterspring_Journey](https://user-images.githubusercontent.com/108195970/175791536-1ffa9b20-2454-4001-83c5-3df8df95cd29.jpg)

## Wowhead link

["Lorax"](https://tbc.wowhead.com/npc=10918/lorax#starts)
["Lorax's Tale"](https://tbc.wowhead.com/quest=5126/loraxs-tale)

## Questie & WoW version

Questie version v6.12.2
Wow Version: Classic TBC.

**Comment by BreakBB** [comment]
Hey @EnterpriseWoW do you have the "Show All Quests below range (Low level quests)" active? Since this is a level 60 quest it shouldn't show on level 70.

For the Journey it seems the quest was not correctly flagged as Blacksmithing quest.

**Comment by EnterpriseWoW** [comment]
"Show all Quests below range (Low level quests)" is enabled.

Hereunder the output of /dump QuestieDB:IsDoable(5126)

```
Dump: value=QuestieDB:IsDoable(5126)
Questie: [SPAM][QuestieDB:IsDoable] class requirement not fulfilled for questId: 5126
[1]=false
```

**Comment by BreakBB** [comment]
Interesting. What class do you have @EnterpriseWoW ?

**Comment by EnterpriseWoW** [comment]
only Shaman :) @BreakBB

**Comment by EnterpriseWoW** [comment]
Hi there, 

I just installed Questie version v6.13.0 but the quest still does not appear as available.
The dump command output also remained the same: 

![image](https://user-images.githubusercontent.com/108195970/180577816-5e9142a2-d625-4b43-ab13-2e3de795c58e.png)

**Comment by BreakBB** [comment]
Thanks for coming back to this @EnterpriseWoW I will have another look

**Comment by Muehe** [comment]
@BreakBB On Wotlk PTR IsDoable returns true now on a 80 Shaman, problem is the character doesn't have blacksmithing. :see_no_evil:

**Comment by BreakBB** [comment]
@Muehe `IsDoable` does not check all things (ignores blacklisting for example). But with the move to Wotlk, we got an entirely new database, so that might have fixed it (for now) 🤷🏻

**Comment by Muehe** [comment]
Nah, the class requirement is missing from all three DBs, and it's loaded from a classic correction.

**Comment by anon1231823** [comment]
I am looking in the classic corrections. We have the class requirement for Shaman + Paladin + Warrior, and requires blacksmithing 270. I think this issue has been resolved. I don't recall other quests having such strange requirements for Class + Profession. And Shaman + Paladin + Warrior is very weird.
